.github/workflows/test_with_woodwork_main_branch.yml

```diff
@@ -0,0 +1,61 @@
+name: Test Featuretools with main Branch
```

Comment:
Add Woodwork to this -> `Test Featuretools with Woodwork main branch`

Reply:
updated